ui/page/transaction/transaction_details_page.go

```diff
@@ -695,6 +730,11 @@ func (pg *TxDetailsPage) Handle() {
 			pg.ChangeFragment(NewTransactionDetailsPage(pg.Load, pg.ticketSpent))
 		}
 	}
+
+	if pg.rebroadcastClicable.Clicked() {
+		pg.rebroadcastClicable.SetEnabled(false, nil)
+		pg.wallet.PublishUnminedTransactions()
+	}
```

Comment:
```suggestion
	if pg.rebroadcastClicable.Clicked() {
		if !pg.Load.WL.MultiWallet.IsConnectedToDecredNetwork() {
			pg.Toast.NotifyError("not connected to the decred network") // there should be a localized string for this already, i'm guessing
			return
		}
		pg.rebroadcastClicable.SetEnabled(false, nil)
		err := pg.wallet.PublishUnminedTransactions()
		if err != nil {
		    //display the error to the user, and if possible log it
		    pg.Toast.NotifyError(err.Error())
		} else {
		    pg.Toast.Notify("Republished unmined transactions to the decred network")
		}
	}
```

Reply:
also the PublishUnminedTransactions(), is a blocking function, so if you notice UI lags/free when spamming the rebroadcast button, it might be better to put the function in a goroutine

```
	if pg.rebroadcastClicable.Clicked() {
		go func() {
			if !pg.Load.WL.MultiWallet.IsConnectedToDecredNetwork() {
				pg.Toast.NotifyError("not connected to the decred network") // there should be a localized string for this already, i'm guessing
				return
			}
			pg.rebroadcastClicable.SetEnabled(false, nil)
			err := pg.wallet.PublishUnminedTransactions()
			if err != nil {
				//display the error to the user, and if possible log it
				pg.Toast.NotifyError(err.Error())
			} else {
				pg.Toast.Notify("Republished unmined transactions to the decred network")
			}
		}()
	}
```